Services/DataBaseService/Commands/CreateUserCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DataBaseService.Interfaces;
+using DTO;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DataBaseService.Commands
+{
+    public class CreateUserCommand : ICommand<UserEmailPassword>
+    {
+        private readonly IRepository<UserEmailPassword> userRepository;
```

Comment:
Can you please explain what is the reason to use repository

Reply:
Repository is an abstraction layer that generalizes basic operations (like create, find, delete and other) with models. For example, adding different models to database can be different.